main.js

```diff
@@ -0,0 +1,99 @@
+const renderer = new Renderer()
+const boardGame = new playerMatrix()
+
+$("#load-board").on("click", function () {
+    boardGame.playersScore.player1Score=0
+    boardGame.playersScore.player2Score=0
+    const rows = $("#rows").val()
+    const columns = $("#columns").val()
+    $("#rows").val("")
+    $("#columns").val("")
+    const coins = Math.floor((rows*columns)/5)
+    const walls = Math.floor((rows*columns)/10)
+    boardGame.loadBoard(rows, columns)
+    boardGame.player1(0, 0, 1)
+    boardGame.player2(rows, columns, 2)
+    boardGame.addingCoins(rows, columns, coins)
+    boardGame.addWalls(rows, columns, walls)
```

Comment:
these lines (defining where the players are, adding coins, etc.) should be in some method in the playerMatrix class, and not here